contents/customers/drivvn.md

```diff
@@ -0,0 +1,72 @@
+
+For example, through experiments in PostHog, they can slowly control releases and test new changes: first, they use PostHog to release the feature with their in-house team at Drivvn. If it functions correctly, they’ll release it externally, to a small audience: say, 10% of visitors. If that’s successful, they’ll roll it out to 50% of users — a safely controlled, step-by-step version of A/B testing that ensures no single, wild change wipes out any of their hard-earned progress.
+
+“It also helps us target a specific brand experience to the right audience,” explains John. Using group analytics, Drivven can identify and aggregate behavior at the company level. “When we recognize a visitor belongs to a B2B company group, we can customize their experience through feature flags targeted at their specific group type.”
+
+But it helps them go wider than that: as an agency, John and his team hold an ecosystem view. It’s not just one website, one audience base. “It’s also across different brands as well — you might release a feature for one brand, but not enable that across the other ones, until you know how well it's going to perform."
+
+It’s here where PostHog’s group analytics feature comes into play yet again. Since groups will help you analyze trends at an entity-level (such as “companies”), as opposed to a user-level, Drivven is empowered to break data apart by brand, by customer type, and any other segment that becomes relevant for them, for better reporting, and ultimately, better outcomes. 
+
+With this birds-eye view, and the ability to run experiments and changes, John’s team can report to their own customers on how well their branded digital experiences are performing. Through dashboards, the reporting area helps them have those conversations — and it’s the data that is changing people’s minds.
+
+“With one customer, I would say they were very against this [experimental, data-driven, A/B approach,” says John. “We then did one small-ish change — as subtle as changing the CTA on one page. When we showed the benefit of that, suddenly it was like a lightbulb moment. It clicked: ‘I understand the benefit of that now.’  After that, they were very keen to continue doing that approach.”
+
+<BorderWrapper>
+<Quote
+    imageSource="/images/customers/johny_laramy.jpg"
+    size="md"
+    name="John Laramy"
+    title="Head of Product Engineering, Drivvn"
+    quote={`“We’ve tried FullStory, MixPanel, Google Analytics and Plausible - but after building out prototypes, trying each tool, PostHog is where I want to be.”`}
+/>
+</BorderWrapper>
+
+“One of the reasons we went with PostHog was because of the wide range of features,” says John. “We're using analytics heavily today across all those brand experiences. We're doing lots of feature flags, and also experiments…all towards helping people get to a clear outcome.”
```

Comment:
```suggestion
“One of the reasons we went with PostHog was because of the wide range of features,” says John. “We're using analytics heavily today across all those brand experiences. We're doing lots of feature flags, and also experiments - all towards helping people get to a clear outcome.”
```